schemas/test/en/test_new_variants_question.json

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
Section title probably missing in this one, will display as 'None' on summary page.